#! /bin/sh /usr/share/dpatch/dpatch-run
## 90_dvdswitch-0.1.3-1.5.4.dpatch by Thomas Günther <tom@toms-cafe.de>
## http://toms-cafe.de/vdr/download/dvdswitch-0.1.3-1.5.4.diff
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Changes for VDR >= 1.5.4.

@DPATCH@
--- dvdswitch-0.1.3/helpers.h
+++ dvdswitch-0.1.3/helpers.h
@@ -80,7 +80,7 @@ class cStringListItem : public cListObje
     char *Value(void) { return String; }
 };
 
-class cStringList : public cList<cStringListItem> {};
+//class cStringList : public cList<cStringListItem> {};
 
 // --- cTokenizer ---------------------------------------
 
